webcomponents/tool/src/main/frontend/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "js/sakai-element.js",
   "workspaces": [ "./packages/*" ],
   "scripts": {
-    "lint": "eslint --fix \"packages/**/src/*.js\"",
+    "lint": "eslint \"packages/**/src/*.js\"",
```

Comment:
```suggestion
    "lint": "eslint \"packages/**/src/*.js\"",
    "lintfix": "eslint --fix \"packages/**/src/*.js\"",
```

Reply:
If we update the pom to call lintfix, I'm cool with this :)

Reply:
then you're modifying files on build which just seems odd to me .....

Reply:
Maybe. I never see it because the pre commit hook does the job in advance.